Src/DependencyCollector/Shared/Implementation/ProfilerSqlProcessingBase.cs

```diff
@@ -92,6 +92,24 @@ public object OnEndForOneParameter(object context, object returnValue, object th
         }
 
         /// <summary>
+        /// On end async callback for methods with 1 parameter.
+        /// </summary>
+        public object OnEndAsyncForOneParameter(object context, object returnValue, object thisObj)
```

Comment:
Who is calling this? Decorator only adds callback for Begin and OnException right?